### PR TITLE
test: integration tests for sending SMS or email test messages

### PR DESCRIPTION
**Pull request by IceHe**
## Summary

Integration tests: 

- Send SMS or email test messages successfully.
- Fail to send SMS or email test messages due to the wrong config.

## Testing

Ditto.

### CODE REVIEW

File: packages/integration-tests/tests/connectors.test.ts [hunk 1]
```diff
@@ -164,9 +166,26 @@ test('connector flow', async () => {
...
+    ).resolves.not.toThrow();
+  });
 
-  // Next up
-  // - validate `config` parameter before sending
-  // - send sms test message
-  // - send email test message
+  it('should fail to send the test message when the config is invalid', async () => {
+    await expect(
+      sendSmsTestMessage(mockSmsConnectorId, '8612345678901', mockEmailConnectorConfig)
```
Comment (darcyYe) [inline packages/integration-tests/tests/connectors.test.ts:183]: Maybe using an empty `config` is good enough for the testing.

**Review by darcyYe — APPROVED** [review]
Overall LGTM

File: packages/integration-tests/tests/connectors.test.ts [hunk 2]
```diff
@@ -164,9 +166,26 @@ test('connector flow', async () => {
...
+    ).rejects.toThrow(HTTPError);
+    await expect(
+      sendEmailTestMessage(mockEmailConnectorId, 'test@example.com', mockSmsConnectorConfig)
```
Comment (darcyYe) [inline packages/integration-tests/tests/connectors.test.ts:186]: ditto

**Comment by IceHe** [comment]
Rebased master to resolve conflicts.

**Pull request merged**